### PR TITLE
TOF: definition of two configurable thresholds in the checker and new shifter message

### PR DESCRIPTION
**Pull request by sofiatomassini**
two different configurable thresholds are added in the checker. 
The shifter message is modified 
The HitMapNoiseFiltered id used as monitoring object in the checker and in the PostProcessing

### CODE REVIEW

**Review by njacazio — APPROVED** [review]

**Review by njacazio — APPROVED** [review]

**Comment by njacazio** [comment]
@Barthelemy ci looks spurious again

**Comment by sofiatomassini** [comment]
@Barthelemy  @knopers8 could you have a look? this seems to be spurious, it would be important to have this merged for the next release. thank you!

File: Modules/TOF/src/CheckHitMap.cxx [hunk 1]
```diff
@@ -136,17 +136,17 @@ void CheckHitMap::beautify(std::shared_ptr<MonitorObject> mo, Quality checkResul
     ILOG(Error, Support) << "Cannot beautify MO " << mo->getName() << " which is not of type " << getAcceptedType() << ENDM;
     return;
   }
-  if (mo->getName() == mAcceptedName) {
+  if (1) {
```
Comment (knopers8) [inline Modules/TOF/src/CheckHitMap.cxx:139]: Shouldn't the `if` be removed in this case?

Comment (njacazio) [reply Modules/TOF/src/CheckHitMap.cxx:139]: that's true, I could not reapply the formatting online. Will do at the next iteration. Thanks!

**Pull request merged**